src/15_classes.py

```diff
@@ -1,29 +1,46 @@
 # Make a class LatLon that can be passed parameters `lat` and `lon` to the
 # constructor
 
-# YOUR CODE HERE
+class LatLon():
+    def __init__(self, lat, lon):
+        self.lat = lat
+        self.lon = lon
 
 # Make a class Waypoint that can be passed parameters `name`, `lat`, and `lon` to the
 # constructor. It should inherit from LatLon. Look up the `super` method.
 
-# YOUR CODE HERE
+class Waypoint(LatLon):
+    def __init__(self, name, lat, lon):
+        self.name = name
+        super().__init__(lat, lon)
```

Comment:
Flip line 14 and 15 with each other. Even if this works, this is not usually the convention.